lib/ReportDataCollector.php

```diff
@@ -399,6 +400,19 @@ private function getOCTablesArray(): array {
 		return $ocTables;
 	}
 
+	public const ETC_OS_RELEASE = '/etc/os-release';
+	public const ETC_LSB_RELEASE = '/etc/lsb-release';
+
+	private function getLinuxDistribution(): string {
+		if (file_exists(self::ETC_OS_RELEASE)) {
+			return file_get_contents(self::ETC_OS_RELEASE);
```

Comment:
Maybe check the return of file_get_contents could return false if the file is not readable. Check that also and return empty string or "Unknown".